titan/features/haart.py

```diff
@@ -57,8 +57,8 @@ def init_agent(self, pop: "population.Population", time: int):
             .drug_type[self.agent.drug_type]
         )
         if (
-            self.agent.hiv
-            and self.agent.hiv_dx
+            self.agent.hiv.active  # type: ignore[attr-defined]
```

Comment:
Is there any way to make mypy hate this less other than the ignore?

Reply:
not that I've figured out.  At least not without hardcoding all the features/things on the agent side, which I also don't want to do.